src/components/Button/index.tsx

```diff
@@ -397,9 +404,11 @@ const FwdButton = ({
         {...props}
         href={href}
         to={to}
+        aria-label={ariaLabel}
         download={download}
         ref={innerRef}
         as={as}
+        role={as === 'a' ? 'button' : undefined}
```

Comment:
Well, if it's a link it's not a button 😄 

```suggestion
        role={as === 'a' ? 'link' : undefined}
```